doc/extension_store/manifest.jsonc

```diff
@@ -0,0 +1,153 @@
+{
+    // Info displayed in the Extension Store
+    "id": "postgres-vs",
+    "name": "PostgreSQL Virtual Schema",
+    "description": "...",
+    "version": "2.0.1",
+    // Dependencies
+    "bucketFsUpload": {
+        "jdbcDriver": {
+            // Download URL for the JDBC driver, e.g. Maven Central URL
+            "downloadUrl": "https://jdbc.postgresql.org/download/postgresql-42.3.3.jar",
```

Comment:
What is, when the cluster doesn't allow internet access? Should we maybe bundle the extensions into a single file.

Reply:
I guess in this case you need to run the extension installer from a machine that has access to both the cluster and the internet.